ios-trivia-game/ViewControllers/InviteViewController.swift

```diff
@@ -41,7 +48,49 @@ class InviteViewController: UIViewController {
         self.inviteTableView.dataSource = self
         self.inviteTableView.estimatedRowHeight = 100
         self.inviteTableView.rowHeight = UITableViewAutomaticDimension
+        
+        refreshInvites()
+        
+        self.inviteTableView.addSubview(self.refreshControl)
+    }
 
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func onTapInviteTableViewCell(_ sender: UITapGestureRecognizer) {
```

Comment:
i was actually using that initially.  I was just trying out some different things.
I could move it back to didSelectRow